### ISSUE TITLE
Ideas for Live Tiles

### ISSUE THREAD

**Issue by Amillennialist** [microsoft/microsoft-ui-xaml#2142]
-Keep Live Tiles.

-Keep future versions compatible with current Tiles. (Editable by programs like EdgeTile Creators.)

-Create an official Microsoft tool (PowerToy?) for creating, backing up, and restoring custom Tiles and layouts.

-Restore 8.x's horizontally scrolling Start as an option (especially for Tablet Mode).

**Comment by StephenLPeters** [comment]
duplicate of #1958

**Comment by Amillennialist** [comment]
That one's closed. Open it so I can comment.

From Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ranjeshj** [comment]
@Amillennialist you can comment on closed threads.  Note that live tiles is not something that is related to or actionable in this repo and the shell team does not actively look at issues here. That particular issue has been forwarded to the shell team, so please comment on that thread. Also, the shell team asked for upvotes in feedback hub that they actively monitor.

**Comment by mdtauk** [comment]
> @Amillennialist you can comment on closed threads. Note that live tiles is not something that is related to or actionable in this repo and the shell team does not actively look at issues here. That particular issue has been forwarded to the shell team, so please comment on that thread. Also, the shell team asked for upvotes in feedback hub that they actively monitor.

The WinUI SDK could include samples and tools to make Live Tiles easier to create and include.  WinUI Desktop apps with customised Live Tiles could be a selling point/feature...

**Comment by ranjeshj** [comment]
@mdtauk are we talking about tiles to show in the app or in the start menu ? I was thinking this issue was talking about the start menu.

**Comment by mdtauk** [comment]
> @mdtauk are we talking about tiles to show in the app or in the start menu ? I was thinking this issue was talking about the start menu.

Yes in the Start Menu (but could also be extended to the Desktop, for something like Windows 10X and Windows 10 I guess)

And the defining and updating of the Live Tile is beyond the scope of WinUI.

However WinUI will be responsible for documentation, and "selling" the new WinUI Desktop app type.  So ensuring WinUI Desktop apps can include an updating Tile, should not be buried. Especially as the Live Tile UI is due to be updated.

![live-tile-changes-2020](https://user-images.githubusercontent.com/7389110/77206922-0a8e1380-6af0-11ea-829d-a74ac9eaedf1.gif)

**Comment by shaheedmalik** [comment]
Live Tiles are something that WinUI handles correct? This is why UWP apps have 'Wide' and 'Large' tile options and Win32 programs don't.

**Comment by contextfree** [comment]
> Live Tiles are something that WinUI handles correct? This is why UWP apps have 'Wide' and 'Large' tile options and Win32 programs don't.

AFAIK this tile support comes with adopting the UWP app model (or packaging your win32 app with MSIX) and don't have much to do with the UI framework as such.
